releasenotes/notes/fix-build-failure-on-32bit-armv7-5b5b36c2d9f9ff00.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+----
```

Comment:
the `reno lint` CI step is failing because this yaml file is invalid, I think because of the extra dash here (should only have 3 dashes)